src/secp256k1.c

```diff
@@ -144,8 +146,12 @@ secp256k1_context* secp256k1_context_preallocated_create(void* prealloc, unsigne
 }
 
 secp256k1_context* secp256k1_context_create(unsigned int flags) {
-    size_t const prealloc_size = secp256k1_context_preallocated_size(flags);
-    secp256k1_context* ctx = (secp256k1_context*)checked_malloc(&default_error_callback, prealloc_size);
+    size_t prealloc_size;
+    secp256k1_context* ctx;
+
+    CHECK(secp256k1_selftest());
```

Comment:
`CHECK` is intended to be used in the tests only. It calls `abort()` (hardcoded) instead of the error callback, so can't even override this on compile time.